components/coding/Expandable.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+
+import { PlusCircleIcon } from "@heroicons/react/solid";
+import { MinusCircleIcon } from "@heroicons/react/solid";
+
+interface ExpandableProps {
+  open?: boolean;
```

Comment:
I like what you did here - I wasn't originally thinking about supplying open as a prop, but this way we can set whether a component is expanded by default on the parent page